CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Next Release (Major)
 
+- Library is now thread-safe (yay, its finally here). Closes GitHub Issue (#183)
+  - Initialize an `Client` object with an API key. Optionally set open and read params
+  - Previous classes have been diverted into `Services` and `Models`
+    - All methods (i.e. `create`, `retrieve`, `all`) exist in services, accessed via property of the client (eg: `client.shipment.create()`)
+      - E.g. `bought_shipment = client.shipment.buy(shipment_id, rate)`
+- Beta namespace changed from `easypost.beta.x` to `client.beta_x`
+- References to `Referral` are now `ReferralCustomer` and `referral_customer` to match the API and docs
+- Empty API response functions return nothing now instead of `true`
```

Comment:
issue: this is not true of the Ruby library. They still return true because you cannot return nothing in ruby

Reply:
I already took the ticket for this and discussed with Matt, to which the determination was to leave it how it is with returning true

Reply:
Yup, they are in the right place. Do we want to return `true` for webhook.delete? In V4, it's just return empty object after refresh: https://github.com/EasyPost/easypost-ruby/blob/v4/lib/easypost/webhook.rb#L33

Reply:
After speaking with the team, we have decided to make `delete` function returning `true` as the API response is empty